### ISSUE TITLE
How to fine tune weights of transformer layers

### ISSUE THREAD

**Issue by lamthuy** [explosion/spacy-transformers#163]
Hi, I have a class that wraps a spacy transformer pipeline as follows:
```
class Transformer(nn.Module):

    def __init__(self):
        super(Transformer, self).__init__()
        self.nlp = spacy.load('en_trf_bertbaseuncased_lg')
    
   def forward(self, sentence):
       doc = self.nlp(sentence)
       return torch.Tensor(doc.tensor)

``` 

When I check the parameters of a Transformer object it is empty. Also the doc.tensor is just an array instead of a tensor with requires_grad = True. Is there any way to fine tune the underlying Spacy Transformer models?
Thank you!

**Comment by svlandeg** [comment]
I'm not sure I understand what you're trying to accomplish here.

> Is there any way to fine tune the underlying Spacy Transformer models?

Yes - you can use a `Transformer` as the `tok2vec` layer for any classification component, and the gradients will flow back to the transformer and update its internal weights. We've recently released version `v1.0.0rc0` of this library that works together with spaCy v3, for which a a pre-release is available as [`spacy-nightly`](https://pypi.org/project/spacy-nightly/).

You can find more docs at https://nightly.spacy.io/usage/embeddings-transformers#transformers and https://nightly.spacy.io/api/transformer. 

If you go through the docs - you'll see that the `Transformer` really is just another component in the `nlp` pipeline (instead of kind of the other way around as implied by your code).
